android-json-form-wizard/src/main/java/com/vijay/jsonwizard/widgets/EditTextFactory.java

```diff
@@ -274,4 +279,24 @@ private void addNumericIntegerValidator(JSONObject jsonObject, MaterialEditText
             }
         }
     }
+
+    private void addRelativeNumericIntegerValidator(JSONObject editTextJSONObject, JsonFormFragment formFragment, MaterialEditText editText) throws JSONException {
+        JSONObject numericIntegerObject = editTextJSONObject.optJSONObject(JsonFormConstants.V_NUMERIC_INTEGER);
```

Comment:
Remove this, should be checked independently by the Numeric Integer validator

Reply:
Done.